http/src/main/java/io/hyperfoil/http/steps/ClearHttpCacheAction.java

```diff
@@ -10,7 +10,10 @@
 public class ClearHttpCacheAction implements Action {
    @Override
    public void run(Session session) {
-      HttpCache.get(session).clear();
+      HttpCache httpCache = HttpCache.get(session);
```

Comment:
Could we drop the `ClearHttpCacheAction` if `httpCache` is not enabled?
This would avoid querying the `httpCache` in the session and having this action in as a whole

Reply:
Mmmh IIRC `ClearHttpCacheAction` is a step that the use should manually set in the benchmark, so if the user does not explicitly set that action it should appear at all at runtime.